src/models/ContentItem.ts

```diff
@@ -0,0 +1,10 @@
+export class ContentItem {
+  name: string;
+  /**
+   * Field is excluded from the response when a locale is set
```

Comment:
I think this comment is above the wrong attribute

Reply:
This has been correctly moved to be above localizedNames